.travis.yml

```diff
@@ -5,11 +5,15 @@ python:
 # command to install dependencies
 install:
   - pip install --upgrade --pre tox
+  - |
+    pip install tox
+    if [[ "${TOXENV#py}" != "$TOXENV" ]]; then
+      export TOX_COVERAGE_RUN="coverage run -m"
```

Comment:
At first glance I thought `TOX_EXTRA_DEPS` was a feature of tox itself. Perhaps we should add a PYTEST_ prefix to avoid confusion? While at it, perhaps there is a way to add extra dependencies using some tox feature?

Reply:
Just remembered that we used `_PYTEST_SETUP_SKIP_PLUGGY_DEP` in [setup.py](https://github.com/pytest-dev/pytest/blob/features/setup.py#L49) for this type of "setup/buid only flag used internally".

`_PYTEST_TOX_COVERAGE_RUN`?

Reply:
Makes sense.  Will rename it to `_PYTEST_TOX_COVERAGE_RUN` and `_PYTEST_TOX_EXTRA_DEPS` then?

Reply:
Sounds good, thanks